app/src/main/res/drawable/sunny_day.xml

```diff
@@ -0,0 +1,28 @@
+<vector android:height="32dp" android:viewportHeight="200.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.sunny_day` appears to be unused
 Unused resources make applications larger and slow down builds.